web-page/src/App.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import 'materialize-css/dist/css/materialize.min.css'
 import M from "materialize-css"
 import './App.css'
-import FilterBox from './components/FilterBox'
+import FilterBox from './components/filter-box/index'
```

Comment:
You don't need the index part here.
```suggestion
import FilterBox from './components/filter-box/'
```

Reply:
Actually the '/' at the end is not needed

Reply:
Done!